error/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+caliptra-cfi-lib = { workspace = true, default-features = false, features = ["cfi", "cfi-counter" ] }
```

Comment:
Do you actually need to set the features here? It seems to me you only need to look at the enum, which shouldn't require any features, correct?

Reply:
Correct. Removed the features.